Lib/test/test_uuid.py

```diff
@@ -34,6 +34,18 @@ def get_command_stdout(command, args):
 class BaseTestUUID:
     uuid = None
 
+    def test_nil_uuid(self):
+        self.assertEqual(
+            self.uuid.NIL,
+            self.uuid.UUID('00000000-0000-0000-0000-000000000000'),
+        )
```

Comment:
```suggestion
        nil = self.uuid.NIL

        s = "00000000-0000-0000-0000-000000000000"
        self.assertEqual(nil, self.uuid.UUID(s))
        self.assertEqual(nil, self.uuid.UUID(int=0))
        self.assertEqual(nil.int, 0)
        self.assertEqual(str(nil), s)
```

And do the same for the MAX.

Reply:
Is this really necessary? Surely the relationship between the string and integer value is already tested effectively elsewhere?

Reply:
I don't know, but since those are dedicated tests, we can do it. Otherwise, we're not really checking something (I mean, NIL is *exactly* `self.uuid.UUID('00000000-0000-0000-0000-000000000000')`, so we're more checking that two constructors calls are equal rather than checking what NIL is meant to represent (namely the 0 uuid)).

Reply:
I know what you mean, but I figured the test is just checking that the constant doesn't get changed to something incorrect. Will update it anyway.